src/applications/mhv-medical-records/components/RecordList/VitalListItem.jsx

```diff
@@ -8,6 +8,19 @@ const VitalListItem = props => {
   const { record } = props;
   const displayName = vitalTypeDisplayNames[record.type];
 
+  const updatedRecordType = (() => {
+    if (record.type === 'PULSE') {
+      return 'HEART-RATE';
+    }
+    if (record.type === 'RESPIRATION') {
+      return 'BREATHING-RATE';
+    }
+    if (record.type === 'PULSE_OXIMETRY') {
+      return 'BLOOD-OXYGEN-LEVEL';
+    }
+    return record.type;
+  })();
```

Comment:
For your two update functions, I think it might make more sense to use a `useMemo` hook instead of an IFFE. This code seems to work fine, but I think `useMemo` makes more sense in a React context.